lib/removeUnusedElements.js

```diff
@@ -12,20 +12,28 @@ module.exports = removeUnusedElements;
  * This function currently only works for accessors, buffers, and bufferViews.
```

Comment:
Some overall comments for this file:

* Avoid any features above es5 because certain files of gltf-pipeline including this one are bundled in CesiumJS which still supports IE11. It's an annoying limitation that isn't made very clear to new contributors, so sorry about that. `const` and `let` are fine though and will work once https://github.com/AnalyticalGraphicsInc/gltf-pipeline/pull/471 is merged. A summary of the bundling situation is here: https://github.com/AnalyticalGraphicsInc/gltf-pipeline/issues/330.
* For style consistency, change `if(` to `if (` everywhere.
* Also style related, use `//` for single line comments rather than `/*  */`

Reply:
> bundled in CesiumJS which still supports IE11

Ah! I was very curious about why you don't use some of the newer features. I suspect in the tests they are ok, cause they are not bundled, right?

Reply:
Yeah, exactly.

Reply:
Alright, I changed:

* `if(` to `if (`
* `x => {}` to `function(x) {}`
* `/* */` to `//`
* default parameter of removeUnusedElements

I hope the two were all the ES6 functions I used (Went through [this resource](http://es6-features.org/) to find them)